src/controllers/changePasswordController.ts

```diff
@@ -0,0 +1,75 @@
+import { Request, Response, NextFunction } from 'express';
+import bcrypt from 'bcrypt';
+import passport from 'passport';
+import Database from '../database';
+import { sendResponse } from '../utils';
+import defineJwtStrategy from '../helper/generalStrategy';
+
+/// ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+interface User {
+  id: string;
+  email: string;
+  password: string;
+}
+
+passport.use('jwt', defineJwtStrategy);
+
+/// ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+const updatePassword = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  passport.authenticate(
+    'jwt',
+    { session: false },
+    async (err: Error | null, user: User) => {
+      try {
+        // if (err) {
+        //   return sendResponse<null>(res, 400, null, 'user not found');
+        // }
+        if (user) {
+          const { oldPassword, newPassword } = req.body;
```

Comment:
same here remove those stategy put it in app.ts and first create it's files you use it in global

Reply:
I have created generalStrategy.ts in helper folder and I made changes.